cumulusci.yml

```diff
@@ -875,21 +875,42 @@ flows:
                 # Wait until after the data migration job has completed
                 options:
                     class_name: RD2_DataMigration_BATCH
-            7:
-                task: execute_anon
-                # Execute the nightly job to update the RD records as needed
-                options:
-                    apex: >
-                        %%%NAMESPACE%%%RD2_OpportunityEvaluation_BATCH rdBatchJob = new %%%NAMESPACE%%%RD2_OpportunityEvaluation_BATCH();
-                        Database.executeBatch(rdBatchJob, rdBatchJob.batchSize);
+#            7:
```

Comment:
Since step 7 is not used, should we just remove it?

Reply:
👍